yarn-project/simulator/src/avm/avm_memory_types.test.ts

```diff
@@ -98,8 +99,105 @@ describe('MeteredTaggedMemory', () => {
   });
 });
 
-type IntegralClass = typeof Uint8 | typeof Uint16 | typeof Uint32 | typeof Uint64 | typeof Uint128;
-describe.each([Uint8, Uint16, Uint32, Uint64, Uint128])('Integral Types', (clsValue: IntegralClass) => {
+type IntegralClass = typeof Uint1 | typeof Uint8 | typeof Uint16 | typeof Uint32 | typeof Uint64 | typeof Uint128;
+
+describe.each([Uint1])('Integral Types (U1 only)', (clsValue: IntegralClass) => {
+  describe(`${clsValue.name}`, () => {
+    it(`Should construct a new ${clsValue.name} from a number`, () => {
+      const x = new clsValue(1);
+      expect(x.toBigInt()).toStrictEqual(1n);
+    });
```

Comment:
Not sure it's worth spending the effort at this time to combine this with the test cases for non-U1 types since they use values other than 0 and 1.